daycare/test_daycare.py

```diff
@@ -61,3 +63,30 @@ def test_daycare_user_creation_with_same_user_name(self, create_daycare_user):
             DayCare.create(username='testUser01', email="valid@gmail.com", password='pass123', name='daycare',
                            description='new description', price_per_day=10, capacity=50,
                            area='north', city='haifa', address='new address')
+
+    def test_daycare_redirected_homepage_after_login(self, client, create_daycare_user):
+        client.force_login(user=create_daycare_user.user)
+        response = client.get('/')
+        assert response.status_code == 302
+        assert response['Location'] == '/profile/'
+
+    def test_relevant_daycare_homepage_with_static_data(self, client, create_daycare_user):
+        dayCare = DayCare.objects.get(pk=1)
+        client.force_login(user=dayCare.user)
+
+        response = client.get('/profile/')
+
+        assert response.context['daycare'] == dayCare
+        assert set(response.context['images']) == set(Image.get_images_by_daycare_id(daycare_id=dayCare.id))
+        assert set(response.context['reviews']) == set(Review.get_review_by_daycare_id(daycare_id=dayCare.id))
+
+    def test_relevant_daycare_homepage_without_static_data(self, client, create_daycare_user):
```

Comment:
I don't see a real need for both a static_data and non_static_data test. 
if anything, I'd prefer to keep the latter. 
WDYT?

Reply:
I agree with @Omeramsc in that case, in the end, we test for some user, so there is no need to separate the test.
@Yuval-Vino because you implemented these tests I'll leave it to you to take care of it, or if you agree with omer too, I can handle it if you want.
 your choice :)